src/ape/api/address.py

```diff
@@ -9,10 +9,23 @@
 
 @abstractdataclass
 class AddressAPI:
+    """
+    A base address API class.
```

Comment:
Add something like "...all account types subclass this type (Externally-Owned Accounts and Contract Accounts)"